README.md

```diff
@@ -166,14 +163,26 @@ See [schema](schema) for the detailed JSON schema for tiles.json.
 
 See the [Q&A below](#Will-tiles.json-be-part-of-the-final-3D-Tiles-spec) for how tiles.json will scale to a massive number of tiles. 
 
-<a name="Creating-Spatial-Data-Structures" />
+### Bounding volume spatial coherence
+
+As described above, the tree has spatial coherence; each tile has a bounding volume completely enclosing its contents, and the content for child tiles are completely inside the parent's bounding volume.  This does not imply that a child's bounding volume is completely inside its parent's bounding volume.  For example:
+
+<p align="center">
+  <img src="figures/parentBoundingSphere.jpg" /><br />
+  Bounding sphere for a terrain tile.
+</p>
+
+<p align="center">
+  <img src="figures/childBoundingSphere.jpg" /><br />
+  Bounding spheres for the four child tiles.  The children's content are completely inside the parent's bounding volume, but the children's bounding volumes are not since they are not tightly fit.
+</p>
+
 ### Creating spatial data structures
 
 The tree defined in tiles.json by `root` and, recursively, its `children`, can define different types of spatial data structures.  In addition, any combination of tile formats and refinement approach (replacement or additive) can be used, enabling a lot of flexibility to support heterogeneous datasets.
 
-It is up to the conversion tool that generates tiles.json to define an optimal tree for the dataset.  A runtime engine, such as Cesium, is generic and will render any tree defined by tiles.json.  Here's a brief descriptions of how 3D Tiles can represent various spatial data structures.
+It is up to the conversion tool that generates tiles.json to define an optimal tree for the dataset.  A runtime engine, such as Cesium, is generic and will render any tree defined by tiles.json.  Here's a brief descriptions of how 3D Tiles can represent various spatial data structures.  This is allowed, but is suboptimal.
 
```

Comment:
The giant bounding spheres.  I can just remove it if you want.  We don't need to over tutorialize this.